web-animations-2/Overview.bs

```diff
@@ -144,9 +144,48 @@ Along with the following updated description:
 </div>
 
 
+<h3 id="timelines">Timelines</h3>
+
+Add:
+
+> The <dfn lt="timeline duration">duration</dfn> of a timeline gives the
+> maximum value a timeline may generate for its current time. This value is
+> used to calculate the [=intrinsic iteration duration=] for the target effect
+> of an animation that is associated with the timeline when the effect's
+> [=iteration duration=] is 'auto'. The value is computed such that the effect
+> fills the available time.  For a monotonic timeline, there is no upper bound
+> on current time, and [=timeline duration=] is unresolved. For a progress based
+> timeline, the [=timeline duration=] is 100%.
+
 <h3 id="animations">Animations</h3>
 
-<div class="informative-bg"><em>This section is non-normative</em>
+Add:
+
+> Animation effects associated with a progress-based timeline require their
+> timing properties to be converted to proportions. The procedure for converting
+> a <dfn lt="time-based animation to a proportional animation"></dnf> is as
```

Comment:
The syntax here appears to be incorrect.

Reply:
Fixed